airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/StandardNaming.java

```diff
@@ -22,12 +22,12 @@
  * SOFTWARE.
  */
 
-package io.airbyte.integrations.base;
+package io.airbyte.integrations.destination;
 
 import io.airbyte.commons.text.Names;
 import java.time.Instant;
 
-public class StandardSQLNaming implements SQLNamingResolvable {
+public class StandardNaming implements IdentifierNamingResolvable {
```

Comment:
StandardNameTransformer? Or StandardNamer (not sure it is possible to use Namer to mean smth that gives name)